docs/fundamentals/code-analysis/quality-rules/ca2255.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "CA2255: The `ModuleInitializer` attribute should not be used in libraries"
```

Comment:
I'm not sure backticks are okay for titles.

Reply:
I saw backticks in several other titles. Example: https://github.com/dotnet/docs/blob/main/docs/fundamentals/code-analysis/quality-rules/ca2251.md.

@IEvangelist?

Reply:
Those should be fixed too, it's okay to have backticks in headings, as that is properly rendered in HTML. But we're talking about the actual `<meta>` tag values at this point. Those just add noise and render like this:

![image](https://user-images.githubusercontent.com/7679720/142046119-31bf7394-21af-438e-8348-e116f3ee3679.png)

Does that make sense?